### PR TITLE
Add "Raman spatial dimensionality"

### LINKED ISSUE #28: [NTR] add "method"
For new term requests, please provide the following information:

## Preferred term label

Raman spatial method

## Synonyms

method

## Textual definition

The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Single spectrum"; "Line scan/mapping"; "Vertical mapping"; "2D mapping"; "3D mapping".

## Suggested parent term

Please look in the hierarchy in a browser such as the [NFDI4Chem TS](https://terminology.nfdi4chem.de/ts/ontologies/vibso)

## Examples

Please provide a more detailed description that examplifies how the term is supposed to be used, in terms of what it is suppose to repesent (e.g. a concept or a relation).

## Attribution

ORCID:0000-0001-7694-5519

### PR DESCRIPTION
**Pull request by Zack-83**
Closes #28

### CODE REVIEW

File: src/templates/vibso_terms.tsv [hunk 1]
```diff
@@ -1,6 +1,7 @@
 id	label	definition	definition source URI	type	super class	super property	inverse property	domain (if property)	range (if property)	term creaor URI (e.g. ORCID)	term curator URI (e.g. ORCID)	example of usage	editor note
 ID	AL rdfs:label@en	AL IAO:0000115@en	>AI dc11:source	TYPE	SC % SPLIT=|	SP %	IP %	DOMAIN	RANGE	AI dc11:creator	AI IAO:0000117	AL IAO:0000112@en	AL IAO:0000116@en
-obo:VIBSO_0000000	vibrational spectrometer configuration specification	A setting datum that specifies the needed settings of a spectrometer used in vibrational spectroscopy.		owl:Class	setting datum 					ORCID:0000-0002-1595-3213			
+obo:tbd	Raman spatial dimensionality	The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Single spectrum"; "Line scan/mapping"; "Vertical mapping"; "2D mapping"; "3D mapping".		owl:Class	method					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955
```
Comment (Zack-83) [inline src/templates/vibso_terms.tsv:3]:
```suggestion
obo:tbd	Raman spatial dimensionality	The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Single spectrum"; "Line mapping"; "Vertical mapping"; "2D mapping"; "3D mapping".		owl:Class	method					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955		:experimentXY :has :RamanSpatialDimensionality . :RamanSpatialDimensionality :is "3D mapping" .	
```

File: src/templates/vibso_terms.tsv [hunk 2]
```diff
@@ -1,6 +1,7 @@
 id	label	definition	definition source URI	type	super class	super property	inverse property	domain (if property)	range (if property)	term creaor URI (e.g. ORCID)	term curator URI (e.g. ORCID)	example of usage	editor note
 ID	AL rdfs:label@en	AL IAO:0000115@en	>AI dc11:source	TYPE	SC % SPLIT=|	SP %	IP %	DOMAIN	RANGE	AI dc11:creator	AI IAO:0000117	AL IAO:0000112@en	AL IAO:0000116@en
-obo:VIBSO_0000000	vibrational spectrometer configuration specification	A setting datum that specifies the needed settings of a spectrometer used in vibrational spectroscopy.		owl:Class	setting datum 					ORCID:0000-0002-1595-3213			
+obo:tbd	Raman spatial dimensionality	The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Single spectrum"; "Line mapping"; "Vertical mapping"; "2D mapping"; "3D mapping".		owl:Class	method					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955		:experimentXY :has :RamanSpatialDimensionality . :RamanSpatialDimensionality :is "3D mapping" .
```
Comment (Zack-83) [inline src/templates/vibso_terms.tsv:3]:
```suggestion
obo:tbd	Raman spatial dimensionality	The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Single spectrum"; "Line mapping"; "Vertical mapping"; "2D mapping"; "3D mapping".		owl:Class	method					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955		:experimentXY :has :RamanSpatialDimensionality . :RamanSpatialDimensionality :is "2D mapping" .	
```

File: src/templates/vibso_terms.tsv [hunk 3]
```diff
@@ -1,6 +1,7 @@
 id	label	definition	definition source URI	type	super class	super property	inverse property	domain (if property)	range (if property)	term creaor URI (e.g. ORCID)	term curator URI (e.g. ORCID)	example of usage	editor note
 ID	AL rdfs:label@en	AL IAO:0000115@en	>AI dc11:source	TYPE	SC % SPLIT=|	SP %	IP %	DOMAIN	RANGE	AI dc11:creator	AI IAO:0000117	AL IAO:0000112@en	AL IAO:0000116@en
-obo:VIBSO_0000000	vibrational spectrometer configuration specification	A setting datum that specifies the needed settings of a spectrometer used in vibrational spectroscopy.		owl:Class	setting datum 					ORCID:0000-0002-1595-3213			
+obo:tbd	Raman spatial dimensionality	The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Single spectrum"; "Line mapping"; "Vertical mapping"; "2D mapping"; "3D mapping".		owl:Class	method					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955		:experimentXY :has :RamanSpatialDimensionality . :RamanSpatialDimensionality :is "2D mapping" .
```
Comment (RainerStosch) [inline src/templates/vibso_terms.tsv:3]:
```suggestion
obo:tbd	Raman spatial dimensionality	The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Point"; "1D horizontal"; "1D vertical"; "2D horizontal"; "2D vertical"; "3D".		owl:Class	method					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955		:experimentXY :has :RamanSpatialDimensionality . :RamanSpatialDimensionality :is "2D mapping" .	
```

added "2D vertical", names coherently chosen

**Comment by Zack-83** [comment]
Hallo @RainerStosch & @StroemPhi ! Ich habe endlich mal Begriffe und Definitionen konform zu unseren letzten Beschlüssen angepasst, manchmal mit einer Prise Fantasie. Wollt ihr mal kontrollieren?

File: src/templates/vibso_classes.tsv [hunk 4]
```diff
@@ -30,8 +30,7 @@ obo:VIBSO_0000023	point distance Y	sampling pace in the Y direction|sampling ste
...
-Together with the focal point distance, this setting defines the region of interest depth."		owl:Class	setting datum	ORCID:0000-0001-7694-5519|ORCID:0000-0002-2239-3956	10	number of measurements Z = (region of interest depth) / (point distance Z)				
+obo:VIBSO_0000027	number of measurements Z	number of points in the Z direction|number of layers per acquisition	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/32	"A setting datum that specifies how many equally configured measurements of a sample are to be performed in the vertical direction (on the Z-axis) of a certain region of interest.Together with the focal point distance, this setting defines the region of interest depth."		owl:Class	setting datum	ORCID:0000-0001-7694-5519|ORCID:0000-0002-2239-3956	10	number of measurements Z = (region of interest depth) / (point distance Z)
```
Comment (Zack-83) [inline src/templates/vibso_classes.tsv:33]:
```suggestion
obo:VIBSO_0000027	number of measurements Z	number of points in the Z direction|number of layers per acquisition	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/32	"A setting datum that specifies how many equally configured measurements of a sample are to be performed in the vertical direction (on the Z-axis) of a certain region of interest. Together with the focal point distance, this setting defines the region of interest depth."		owl:Class	setting datum	ORCID:0000-0001-7694-5519|ORCID:0000-0002-2239-3956	10	number of measurements Z = (region of interest depth) / (point distance Z)				
```

**Comment by StroemPhi** [comment]
Arrgh. I thought I could just rename the branch, as "Zack-83-patch-1" was too unspecific for getting back to VIBSO work after my forced long break. But maybe this is not a bad thing and we can start with a cleaner PR.

**Pull request closed without merging**